File: BackEnd/ajax/AtencionPreHospitalaria.js
$.ajax({
  url:CONST_SITIO_URL+'/services/AtencionPreHospitalaria.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'crear',
    'observacion': null,
    'idAmbulancia': null,
    'idParamedico': null,
    'idExpediente': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/AtencionPreHospitalaria.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorParamedico',
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/AtencionPreHospitalaria.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorCentroDiarias',
    'idCentroMedico': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/AtencionPreHospitalaria.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorCentroFecha',
    'idCentroMedico': null,
    'fechaHoraAtencion': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/AtencionPreHospitalaria.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'actualizar',
    'idAmbulancia': null,
    'observacion': null,
    'idExpediente': null,
    'idAtencion': null,
    'fechaHoraAtencion': null,
    'idParamedico': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/AtencionPreHospitalaria.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorCentroMedico',
    'idCentroMedico': null,
    'nombreCentro': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/AtencionPreHospitalaria.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorPaciente',
    'idExpediente': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});

File: BackEnd/postman/AtencionPreHospitalaria.py
import requests
def postear(data, url):
    return requests.post('http://127.0.0.1/syme/BackEnd/services/'+url+'.php', data=data).content

url='AtencionPreHospitalaria'
data={
  'accion':'crear',
  'observacion': None,
  'idAmbulancia': None,
  'idParamedico': None,
  'idExpediente': None,
}
print postear(data,url)
print

url='AtencionPreHospitalaria'
data={
  'accion':'listarPorParamedico',
}
print postear(data,url)
print

url='AtencionPreHospitalaria'
data={
  'accion':'listarPorCentroFecha',
  'idCentroMedico': None,
  'fechaHoraAtencion': None,
}
print postear(data,url)
print

url='AtencionPreHospitalaria'
data={
  'accion':'actualizar',
  'idAmbulancia': None,
  'observacion': None,
  'idExpediente': None,
  'idAtencion': None,
  'fechaHoraAtencion': None,
  'idParamedico': None,
}
print postear(data,url)
print

url='AtencionPreHospitalaria'
data={
  'accion':'listarPorCentroMedico',
  'idCentroMedico': None,
  'nombreCentro': None,
}
print postear(data,url)
print

url='AtencionPreHospitalaria'
data={
  'accion':'listarPorPaciente',
  'idExpediente': None,
}
print postear(data,url)
print
